### PR TITLE
Upload to aws

### PR DESCRIPTION
**Pull request by b2simms**
1. Updated pom.xml and build.xml with proper names - compiles to **Scheduler.war**
2. AuthenticationFilter: added mobile auth if RequestFromMobile set in header
3. Changed database queries to support new database schema
4. DBVar is pointed to **Production database**; please use _local_ if you can
5. Moved a couple of methods around in Employee
6. index.jsp changed to support partials - dynamically based on employee state
7. Partials created for clocking and breaks - see 6
8. Removed commented/unused authentication code
9. **Static** JSON is used in clockin.js, inshift.js, and breakin.js - Wei is adding it so it is dynamic

### CODE REVIEW

File: src/main/java/org/AuthenticateDbHandler.java [hunk 1]
```diff
@@ -196,6 +196,56 @@ private String getSecretKey() {
     	return null;
 	}
 
+	public Claims getClaims(WebTokens compactJws) {
```
Comment (JoshNorthrup) [inline src/main/java/org/AuthenticateDbHandler.java:199]: this method is almost exactly `isAuth`, is it possible to wrap most of the repeated logic in a private method?

File: src/main/java/org/ClockDbHandler.java [hunk 2]
```diff
@@ -51,9 +52,9 @@ private Employee getEmployeeClockStateandWorkedShiftID(int employee_id, int shif
     	OraclePreparedStatement stmt = null;
```
Comment (JoshNorthrup) [inline src/main/java/org/ClockDbHandler.java:52]: nothing to do with your changes but why do we have this method? I feel like we can just use `Employee.getEmployeeById`

File: src/main/java/org/Employee.java [hunk 3]
```diff
@@ -115,6 +118,29 @@ public void setCurrent_worked_shift_id(int current_worked_shift_id) {
 		this.current_worked_shift_id = current_worked_shift_id;
 	}
 
+	private void setPassword(byte[] salt, byte[] pass){
```
Comment (JoshNorthrup) [inline src/main/java/org/Employee.java:121]: I guess this branch was a bit behind master, can you move the private methods back to the bottom as they are on master?

File: src/main/java/org/AuthenticateDbHandler.java (hunk 1 above)
Comment (b2simms) [reply src/main/java/org/AuthenticateDbHandler.java:199]: Currently, `isAuth` is not being used. I have commented it out of my code.

Comment (b2simms) [reply src/main/java/org/AuthenticateDbHandler.java:199]: Another method, `public int getInt(String key, WebTokens compactJws)` is almost the same as `public Claims getClaims(WebTokens compactJws)`

File: src/main/java/org/ClockDbHandler.java (hunk 2 above)
Comment (b2simms) [reply src/main/java/org/ClockDbHandler.java:52]: This predates the `Employee.getEmployeeById`. It should be refactored.

Comment (b2simms) [reply src/main/java/org/ClockDbHandler.java:52]: I refactored it.

File: src/main/java/org/Employee.java (hunk 3 above)
Comment (b2simms) [reply src/main/java/org/Employee.java:121]: Employee class is the same as master now.

File: src/main/java/org/AuthenticateDbHandler.java (hunk 1 above)
Comment (b2simms) [reply src/main/java/org/AuthenticateDbHandler.java:199]: They do distinctly different operations. I am going to leave them as is.

**Comment by b2simms** [comment]
Refer to new PR `mobile friendly` for the implemented changes.

**Pull request closed without merging**